bootstrap.php

```diff
@@ -20,9 +21,10 @@
 $configurator = new Configurator();
 $configurator->setDebugMode( PACKETERY_DEBUG );
 
+Debugger::$logDirectory = PACKETERY_PLUGIN_DIR . '/log';
```

Comment:
Používá se to i bez PACKETERY_DEBUG a volání enableDebugger ?

Reply:
Ano. Chyby se mají logovat, když k nim dojde.